docs/vision-usage.rst

```diff
@@ -0,0 +1,233 @@
+
+.. code-block:: python
+
+     >>> from gcloud import vision
+     >>> client = vision.Client()
+
+or pass in ``credentials`` and ``project`` explicitly
+
+.. code-block:: python
+
+     >>> from gcloud import vision
+     >>> client = vision.Client(project='my-project', credentials=creds)
+
+Annotating an Image
+-------------------
+
+Annotate a single image
+~~~~~~~~~~~~~~~~~~~~~~~
+
+.. code-block:: python
+
+    >>> from gcloud import vision
+    >>> client = vision.Client()
+    >>> image = client.image('./image.png')
+    >>> faces = image.detect_faces(limit=10)
+
+Annotate multiple images
+~~~~~~~~~~~~~~~~~~~~~~~~
+
+.. code-block:: python
+
+     >>> first_image = client.image('./image.jpg')
+     >>> second_image = client.image('gs://my-storage-bucket/image2.jpg')
+     >>> with client.batch():
+     ...     labels = first_image.detect_labels()
+     ...     faces = second_image.detect_faces(limit=10)
+
+or
+
+.. code-block:: python
+
+     >>> images = []
+     >>> images.append(client.image('./image.jpg'))
+     >>> images.append(client.image('gs://my-storage-bucket/image2.jpg'))
+     >>> faces = client.detect_faces_multi(images, limit=10)
+
+No results returned
+~~~~~~~~~~~~~~~~~~~
+
+Failing annotations return no results for the feature type requested.
+
+.. code-block:: python
+
+    >>> from gcloud import vision
+    >>> client = vision.Client()
+    >>> image = client.image('./image.jpg')
+    >>> logos = image.detect_logos(limit=10)
+    >>> logos
+    []
+
+
+Manual Detection
+~~~~~~~~~~~~~~~~
+
+You can call the detection method manually.
+
+.. code-block:: python
+
+    >>> from gcloud import vision
+    >>> client = vision.Client()
+    >>> image = client.image('gs://my-test-bucket/image.jpg')
+    >>> faces = image.detect(type=vision.FACE_DETECTION, limit=10)
+
+Face Detection
+~~~~~~~~~~~~~~
+
+Detecting a face or faces in an image.
+For a list of the possible facial landmarks
+see: https://cloud.google.com/vision/reference/rest/v1/images/annotate#type_1
+
+
+.. code-block:: python
+
+    >>> from gcloud import vision
+    >>> client = vision.Client()
+    >>> image = client.image('./image.jpg')
+    >>> faces = image.detect_faces(limit=10)
+    >>> faces[0].landmarks[0].type
+    'LEFT_EYE'
+    >>> faces[0].landmarks[0].position.x
+    1301.2404
+    >>> faces[0].detection_confidence
+    0.9863683
+    >>> faces[0].joy_likelihood
+    0.54453093
+    >>> faces[0].anger_likelihood
+    0.02545464
+
+
+
+Label Detection
+~~~~~~~~~~~~~~~
+
+Image labels are a way to help categorize the contents of an image.
+If you have an image with a car, person and a dog it, label detection will
+attempt to identify those objects.
+
+.. code-block:: python
+
+    >>> from gcloud import vision
+    >>> client = vision.Client()
+    >>> image = client.image('./image.jpg')
+    >>> labels = image.detect_labels(limit=3)
+    >>> labels[0].description
+    'automobile'
+    >>> labels[0].score
```

Comment:
So, this is an artifact of the Vision API that we should probably smooth out.
`score` is from [EntityAnnotation](https://cloud.google.com/vision/reference/rest/v1/images/annotate#entityannotation) and `detection_confidence` is from [FaceAnnotation](https://cloud.google.com/vision/reference/rest/v1/images/annotate#faceannotation)